bin/sanctuary-benchmark

```diff
@@ -2,41 +2,47 @@
 
 'use strict';
 
-var fs = require('fs');
+var fs = require ('fs');
 
-var dir = require('path').resolve(process.cwd(), 'bench');
+var path = require ('path');
+
+
+var dir = path.resolve (process.cwd (), 'bench');
 
 var help = 'Create a bench directory in this project to get started\n';
 
 function printHelp() {
-  process.stdout.write(help);
-  process.exit(0);
+  process.stdout.write (help);
+  process.exit (0);
 }
 
-if (!fs.existsSync(dir)) printHelp();
+if (!(fs.existsSync (dir))) printHelp ();
 
-var benchmarks = fs.readdirSync(dir).filter(function(file) {
-  return file.slice(-3) === '.js';
-}).map(function(file) {
-  return file.slice(0, -3);
-});
+var benchmarks = (
+  fs.readdirSync (dir)
+  .filter (function(file) { return file.slice (-3) === '.js'; })
+  .map (function(file) { return file.slice (0, -3); })
+);
 
+/* eslint-disable max-len */
 help = 'There are no benchmarks in the bench directory\n'
      + '\n'
      + '  To create a benchmark, add a JavaScript file to the bench directory\n'
      + '  and define benchmarks as described by the sanctuary-benchmark README.\n'
      + '\n'
      + 'Example\n'
      + '\n'
-     + '  module.exports = require(\'sanctuary-benchmark\')(\n'
-     + '    require(\'ramda\'),\n'
-     + '    require(\'sanctuary\'),\n'
-     + '    { left: \'Ramda\', right: \'Sanctuary\' },\n'
-     + '    { map: (lib) => lib.map(x => x + 1, [1, 2, 3]) }\n'
+     + "  module.exports = require ('sanctuary-benchmark') (\n"
+     + "    require ('ramda'),\n"
+     + "    require ('sanctuary'),\n"
+     + "    {left: 'Ramda', right: 'Sanctuary'},\n"
+     + '    {map: lib => lib.map (x => x + 1) ([1, 2, 3])}\n'
      + '  );\n';
```

Comment:
I reformatted the example in accordance with sanctuary-style, and treated `lib.map` as a curried function. :)